src/odb/src/db/dbSBox.h

```diff
@@ -46,14 +46,25 @@ class dbIStream;
 class dbOStream;
 class dbDiff;
 
-struct _dbSBoxFlags
+struct _dbSBoxFlagsNoMask
 {
   dbWireShapeType::Value _wire_type : 6;
   uint _direction : 2;  // 0 = undefiend, 1 = horizontal, 2 = vertical, 3 =
                         // octilinear
   uint _spare_bits : 24;
 };
 
+struct _dbSBoxFlags
+{
+  dbWireShapeType::Value _wire_type : 6;
+  uint _direction : 2;  // 0 = undefiend, 1 = horizontal, 2 = vertical, 3 =
+                        // octilinear
+  uint _via_bottom_mask : 2;
+  uint _via_cut_mask : 2;
+  uint _via_top_mask : 2;
+  uint _spare_bits : 18;
```

Comment:
I don't think you actually need to rev the schema and add this backward compatibility as you are just converting spare bits to used.  The spare bits are 0 already in old dbs.